code/style.css

```diff
@@ -1,4 +1,140 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: #d8d7d5;
+} 
+
+/* header */
+.header {
+  display: flex;
+  justify-content: space-between;
 }
+
+.logo {
+  width: 100px;
+  margin: 25px;
```

Comment:
The logo appears in different sizes on different browsers. Maybe it can be fixed with percentage and max-width.